### ISSUE TITLE
Add explicit Sec-CH-UA-Brand

### ISSUE THREAD

**Issue by rowan-m** [WICG/ua-client-hints#204]
The spec includes `Sec-CH-UA-FullVersion` which returns an individual value representing the complete version for the user-agent. However there is not a definite way to tie this back to the actual version. Currently, a server could iterate through the entries in `Sec-CH-UA` and attempt to match against the major version to infer the brand, but that's pretty messy and unreliable.

Since the optional high-entropy `Sec-CH-UA-FullVersion` token is already present, I'm suggesting that we add `Sec-CH-UA-Brand` in the same way. So, server would specify:

``` http
  Accept-CH: Sec-CH-UA-Brand, Sec-CH-UA-FullVersion
```

And the user-agent may respond:

``` http
  Sec-CH-UA: "Examplary Browser"; v="73", ";Not?A.Brand"; v="27"
  Sec-CH-UA-Brand: "Examplary Browser"
  Sec-CH-UA-Full-Version: "73.3R8.2H.1"
```

Likewise from the JavaScript side:

```js
navigator.userAgentData
.getHighEntropyValues(["brand", "uaFullVersion"])
.then(ua => { console.log(ua) });
// {
//   "brand": "Examplary Browser",
//   "uaFullVersion": "90.0.4412.3"
// }
```

I think it should be re-emphasised in the spec that this would not be for general compatibility checks, which are served by the GREASEy `Sec-CH-UA` header but rather this covers the need for working around issues with specific versions, reporting, or potentially user-facing information (e.g. "Your last log-in was from Chrome on Android").

**Comment by erik-anderson** [comment]
I would prefer this not happen. With the current greased Sec-CH-UA, it makes it much less likely that someone will erroneously block a smaller market share browser since they will have to generally ignore unknown values.

With a non-greased value, some site developers would likely add a check that says, "if (Sec-CH-UA-Brand != null && !supportedBrowserList.contains(Sec-CH-UA-Brand)) { /* declare non-support */ }" just because they think it's easier to parse/deal with than the greased low-entropy value or they want to scope support only to a fixed set of browsers. As a result, smaller browser vendors would likely be forced to lie.

In other words, I don't believe the benefit of greasing values goes away just because it's not in the included-by-default set.

**Comment by miketaylr** [comment]
I think I agree with @erik-anderson here. But thanks for the suggestion, @rowan-m (we may want to revisit this in the future, not sure).
